### PR TITLE
add jquery-autotab@1.9.2 w/ git auto-update

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #9121 
@pvnr0082t , @x09326 please help me review this pull request, thank you.
Mnually add other versions from git repo because they can't get from npm
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/Mathachew/jquery-autotab
- Official website (optional, not the repository):
- NPM package url (optional): 
- GitHub / Bitbucket popularity (required):
  - Count of stars: 349
  - Count of watchers: 19
  - Count of forks: 83
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month: 
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [ ] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/jquery-autotab/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "files": [
+        "js/jquery.autotab*"
+      ]
+    },
+    {
+      "basePath": "",
+      "files": [
+        "css/screen*"
+      ]
+    }
```
Comment (pvnr0082t) [inline ajax/libs/jquery-autotab/package.json]: @kennynaoh The basePath of these two are the same. Why don't you combine them?

**Comment by pvnr0082t** [comment]
@kennynaoh It seems that the npm package is not maintained officially. I think it should use git auto-update.

**Comment by kennynaoh** [comment]
@pvnr0082t 
Right
I have fixed it.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@kennynaoh Please rebased it on the latest master branch, thanks.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have fixed it and rebased to the latest master branch.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
@kennynaoh There may be some problems. Please trigger CI again, thank you.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I think it's ok now.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
You guys must be kidding about the repo url.

File: ajax/libs/jquery-autotab/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "description": "A jQuery plugin that provides auto tabbing and filtering on text fields in a form",
+  "keywords": [
+    "auto",
+    "tabbing",
+    "filtering",
+    "jquery"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mvasilkov/jquery-autotab.git"
```
Comment (pvnr0082t) [inline ajax/libs/jquery-autotab/package.json]: @kennynaoh Please correct the url, as well as auto-update target.

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have fixed it.
Please help me review it again.
Thanks!

File: ajax/libs/jquery-autotab/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "autotab",
+    "filter"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Mathachew/jquery-autotab.git"
+  },
+  "author": {
+    "name": "Matthew Miller",
+    "web": "https://github.com/Mathachew"
```
Comment (pvnr0082t) [inline ajax/libs/jquery-autotab/package.json]: @kennynaoh Please use `"url"` as key. Ref: https://docs.npmjs.com/files/package.json#people-fields-author-contributors

**Comment by kennynaoh** [comment]
@pvnr0082t 
I have fixed it.
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
LGTM 
ping @PeterDaveHello

**Comment by PeterDaveHello** [comment]
The css file looks like for example, doesn't it?

**Comment by kennynaoh** [comment]
Right, I think we just need `jquery.autotab.js` https://github.com/Mathachew/jquery-autotab#installation
Thanks!

**Comment by kennynaoh** [comment]
@pvnr0082t 
Please help me review it again.
Thanks!

**Comment by pvnr0082t** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Why don't you use its homepage info?

**Comment by kennynaoh** [comment]
@PeterDaveHello , you mean this http://autotab.mathachew.com/?
I think it just for demo, so I didn't use it.
Should I add it?
Thanks!

**Comment by PeterDaveHello** [comment]
Nobody knows what you think if you don't tell us, that's not how we work, please say it before been asked.

**Comment by kennynaoh** [comment]
So should I add it?

**Pull request merged**